web/css/main.css

```diff
@@ -131,22 +131,45 @@
 	content: "\f1d9";
 }
 
+.report_TAF {
+    border-radius: 5px;
+	background-color: cornsilk;
+	color: black;
+}
+
+.report_PIREP {
+    border-radius: 5px;
+	background-color: gainsboro;
+	color: black;
+}
+
+.report_WINDS {
+    border-radius: 5px;
+	background-color: lavender;
+	color: black;
+}
+
 .flight_condition_VFR {
+    border-radius: 5px;
 	background-color: forestgreen;
 	color: white;
 }
 
 .flight_condition_MVFR {
+    left-border: 120px;
```

Comment:
Oops. Get rid of that. It has no effect. I was trying to make the oval a bit larger and that is a leftover